roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/create-sap-bridge.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Create SAP bridge NodeNetworkConfigurationPolicy
+  kubernetes.core.k8s:
+    state: present
+    definition:
```

Comment:
Store `NodeNetworkConfigurationPolicy` definition in a template?

Reply:
While that's technically possible it would involve another step of rendering that template before applying it. I don't see the benefit for now but let's keep that in the back of our mind in case that code get's too convoluted.